### DISCUSSION TITLE
[Feature] Add justified layout to album view

### DISCUSSION THREAD

**Discussion by waclaw66** [immich-app/immich#3602] [Feature Request]
### The feature

Currently album view uses strict grid - matrix. It's not recognizable which asset is landscape or portrait.
It would be nicer to use the already implemented [justified layout](https://github.com/immich-app/immich/pull/2666) for albums as well.  Android version uses that non-matrix view already, but in the simplified way (not nice).

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
Converted to issue to track (#3606)
